l10n_it_declaration_of_intent/demo/declaration_of_intent.xml

```diff
@@ -36,9 +36,13 @@
         ]"
         />
     </record>
-    <record id="l10n_it_esportatore_abituale" model="account.fiscal.position">
+    <record id="l10n_it_esportatore_abituale" model="account.fiscal.position.template">
```

Comment:
Come mai non vogliamo più creare una posizione fiscale ma un template?

In runboat ho attivato entrambe le aziende ma non trovo né questa posizione fiscale:
![image](https://github.com/OCA/l10n-italy/assets/143788994/caeecedf-c9cb-4619-9353-9ddabb2ed62e)
né il partner dichiarato poco sotto:
![image](https://github.com/OCA/l10n-italy/assets/143788994/62c1638a-f881-4269-9341-280ea296e7a5)

Provando ad aggiornare il modulo ottengo
![image](https://github.com/OCA/l10n-italy/assets/143788994/fbb1af69-c7b2-4446-89b0-1cc2ec3e94d2)
e il log non dice nulla di utile
```log
2024-07-09 16:02:19,052 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: Loading module l10n_it_declaration_of_intent (107/164) 
2024-07-09 16:02:19,289 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.registry: module l10n_it_declaration_of_intent: creating or updating database tables 
2024-07-09 16:02:19,481 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/security/declaration_security.xml 
2024-07-09 16:02:19,484 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/security/ir.model.access.csv 
2024-07-09 16:02:19,494 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/data/sequence.xml 
2024-07-09 16:02:19,496 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/wizard/select_declarations_view.xml 
2024-07-09 16:02:19,507 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/views/account_view.xml 
2024-07-09 16:02:19,522 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/views/declaration_of_intent_view.xml 
2024-07-09 16:02:19,552 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/views/company_view.xml 
2024-07-09 16:02:19,565 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: loading l10n_it_declaration_of_intent/views/account_invoice_view.xml 
2024-07-09 16:02:19,631 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.addons.base.models.ir_module: module l10n_it_declaration_of_intent: loading translation file it for language it_IT 
2024-07-09 16:02:19,677 18 INFO b6bea5fc7-7e2a-4bc0-b232-f733bad90cd2 odoo.modules.loading: Module l10n_it_declaration_of_intent loaded in 0.62s, 369 queries (+369 other)
```